core/lib/presentation/utils/html_transformer/html_utils.dart

```diff
@@ -97,6 +97,12 @@ class HtmlUtils {
     </script>
   ''';
 
+  static const lineHeight100Percent = (
+    script: '''
+      document.querySelectorAll("*")
+        .forEach((element) => element.style.lineHeight = "100%");''',
+    name: 'lineHeight100Percent');
```

Comment:
1. We should only query tags with style `lineHeight`.
2. It will affect other tags when it wants to use `lineHeight` of a certain value. So you should only change it for the case `copy & paste` or `drag & drop` text